### PR TITLE
Add Game Server Orchestrator to showcase (team help)

### PR DESCRIPTION
**Pull request by Elliot-Alexander**
Apologies for the previous pull, apparently I can't read which repo I'm in. Anyway I hope these commits are correct.

### CODE REVIEW

File: content/showcase/_index.html [hunk 1]
```diff
@@ -143,6 +143,14 @@ <h1 class="title is-2">UQCS Showcase</h1>
                         image-class="showcase-image"
                         class="showcase-element has-text-centered"
                     >}}
+		    {{< showcase-item 
+                        src="/img/media/showcase/charon.png"
```
Comment (bradleysigma) [inline content/showcase/_index.html:147]: This should be `/img/media/showcase/game_server.png`

File: content/showcase/_index.html [hunk 2]
```diff
@@ -143,6 +143,14 @@ <h1 class="title is-2">UQCS Showcase</h1>
                         image-class="showcase-image"
                         class="showcase-element has-text-centered"
                     >}}
+		    {{< showcase-item
```
Comment (bradleysigma) [inline content/showcase/_index.html:146]: Newest projects should be at the top.

**Review by bradleysigma — CHANGES_REQUESTED** [review]
The showcase images are displayed square. You may want to edit your image to be less long. Also, an image more specific to your project would be better.

**Comment by nicklambourne** [comment]
For reference, showcase images are 1000*750px :)

**Comment by Elliot-Alexander** [comment]
For sure! Will update in a bit and let you know :)

**Comment by Elliot-Alexander** [comment]
Hey :) have updated the pull. Hopefully this is better :D Let me know

**Review by jdcaperon — APPROVED** [review]
LGTM after conflicts

File: content/showcase/_index.html [hunk 3]
```diff
@@ -12,6 +12,14 @@ <h1 class="title is-2">UQCS Showcase</h1>
         <div class="columns is-centered">
             <div class="column is-8">
                 <div class="showcase-gallery columns is-multiline">
+	  	    {{< showcase-item 
+                        src="/img/media/showcase/game_server.png"
```
Comment (bradleysigma) [inline content/showcase/_index.html:16]: Please add an underscore to the image file to match this `src`.

**Review by bradleysigma — DISMISSED** [review]

**Comment by Elliot-Alexander** [comment]
God I am so sorry, I so out of it this week. Should hopefully be fixed up now

**Review by nicklambourne — APPROVED** [review]

**Review by bradleysigma — APPROVED** [review]

**Pull request merged**